packages/forma-36-website/src/components/Promo.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import tokens from '@contentful/forma-36-tokens';
+import { Tag } from '@contentful/forma-36-react-components';
+import { Flex } from '@contentful/forma-36-react-components/dist/alpha';
```

Comment:
We don't need the alpha anymore btw 🥳

Reply:
Or do we? I see you added it elsewhere.

Reply:
So... this is not working then. This is throwing me error when I try to build website.

Reply:
maybe something is wrong on in the website, I will have another look. thx! Forgot that we already have it in the main build.

Reply:
AAAA ok, 🤦‍♀️  I know what's the issue. @denkristoffer thx!

Reply:
✅